pom.xml

```diff
@@ -100,7 +100,7 @@ flexible messaging model and an intuitive client API.</description>
       --add-opens java.management/sun.management=ALL-UNNAMED <!--JvmDefaultGCMetricsLogger-->
     </test.additional.args>
     <testReuseFork>true</testReuseFork>
-    <testForkCount>4</testForkCount>
+    <testForkCount>0</testForkCount>
```

Comment:
this is not related to the pull. please revert

Reply:
Fixed